README.md

```diff
@@ -1,8 +1,19 @@
 # styles
 The billinge group matplotlib style sheet.
 
-To use this style put the following into your plot generating code:
+## Workflow
+1. clone to local drive via `git clone https://github.com/pdfttp/styles.git`
+1. put the following into your plot generating code:
 ```python
 import matplotlib.pyplot as plt
 plt.style.use(os.path.join('/path/to/this/folder', 'billinge.mplstyle'))
 ```
+
+OR
+
+If you are doing more intense work and are expecting to need to change this please:
```

Comment:
Sure, I'll put in a note that everything can be modified inside the python code. If someone is editing a style document I hope it would be in version control.